lca-cli/seedcreator/seedcreator.go

```diff
@@ -302,6 +305,40 @@ func (s *SeedCreator) stopServices() error {
 	return nil
 }
 
+// getVarLibFilelist parses the MCD currentconfig to get the list of managed files under /var/lib
+func (s *SeedCreator) getVarLibFilelist() ([]string, error) {
+	var filelist []string
+	varlibRegex := regexp.MustCompile(`^/var/lib/`)
```

Comment:
why only var/lib ? i believe it is easier but same issue can be with etc/ too

Reply:
The etc tarball is based on the ostree admin config-diff and doesn't (currently) include any broad wildcard exclusions, so wouldn't face the same issue as the var tarball.